Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "languageserver-types"
-version = "0.45.0"
```

Comment:
Please discard this version number change.

It will be updated automatically when releasing.

Reply:
Now it's removed